scripts/exrrfs_ensmean.sh

```diff
@@ -5,6 +5,10 @@ set -x
 
 cpreq=${cpreq:-cpreq}
 cd "${DATA}"  || exit 1
+
+export CMDFILE=$DATA/poescript_ncea
+: > "$CMDFILE"  # Clear or create the CMDFILE
```

Comment:
suggest using the full `${..}` for bash variable names for consistency and robustness.
`$DATA -> ${DATA}`
`$CMDFILE -> ${CMDFILE}`
`$err -> ${err}`